recipes/azure/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "1.0.3" %}
+
+package:
+  name: azure
+  version: {{ version }}
+
+source:
+  fn: azure-{{ version }}.zip
+  url: https://pypi.io/packages/source/a/azure/azure-{{ version }}.zip
+  md5: 0ef1699615c6a4b00a16fdff3cb0569f
+  
+
+build:
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - azure-mgmt ==0.20.1
+    - azure-servicebus ==0.20.1
+    - azure-storage ==0.20.3
+    - azure-servicemanagement-legacy ==0.20.1
+  run:
+    - python
+    - azure-mgmt ==0.20.1
+    - azure-servicebus ==0.20.1
+    - azure-storage ==0.20.3
+    - azure-servicemanagement-legacy ==0.20.1
```

Comment:
Yes, they probably are. One downside though, is that figuring out all the dependencies isn't as straightforward, because they cut everything up into tiny pieces with disparate versions